resources/console/profile-evaluation.yaml

```diff
@@ -0,0 +1,25 @@
+web:
+  console:
+    resources:
+      requests:
+        cpu: 2m
+        memory: 2Mi
+      limits:
+        cpu: 5m
+        memory: 16Mi
+  core-ui:
+    resources:
+      requests:
+        cpu: 2m
+        memory: 16Mi
+      limits:
+        cpu: 10m
+        memory: 32Mi
+backend:
+  resources:
+    requests:
+      cpu: 2m
+      memory: 40Mi
+    limits:
+      cpu: 10m
```

Comment:
I think limit values for CPU are a bit to low, take a look on values of other components (search for "Describe nodes (after installation)"):
https://storage.googleapis.com/kyma-prow-logs/logs/adamwalach_test_of_prowjob_kyma-integration-evaluation-gardener/1334523088292810752/build-log.txt
Maybe we could set it to a value closer to `50m`?
That may improve startup time as well
That comment is about all cpu limits -  for other components as well.